addons/io_hubs_addon/components/operators.py

```diff
@@ -500,6 +502,74 @@ def execute(self, context):
         return {'FINISHED'}
 
 
+class OpenImage(Operator):
+    bl_idname = "image.hubs_open_image"
+    bl_label = "Open Image"
+    bl_options = {'REGISTER', 'UNDO'}
+
+    filepath: StringProperty(subtype="FILE_PATH")
+    files: CollectionProperty(type=PropertyGroup)
+    filter_folder: BoolProperty(default=True, options={"HIDDEN"})
+    filter_image: BoolProperty(default=True, options={"HIDDEN"})
+    target_property: StringProperty()
+
+    relative_path: BoolProperty(
+        name="Relative Path", description="Select the file relative to the blend file", default=True)
+
+    disabled_message = "Can't open/assign images to linked data blocks. Please make it local first"
+
+    @ classmethod
+    def description(cls, context, properties):
+        description_text = "Load an external image "
+        ob = getattr(context, 'host')
+        if bpy.app.version < (3, 0, 0) and is_linked(ob):
+            description_text += f"\nDisabled: {cls.disabled_message}"
+
+        return description_text
+
+    @ classmethod
+    def poll(cls, context):
+        ob = getattr(context, 'host')
+        if is_linked(ob):
```

Comment:
Pretty much the same as above, except the poll only fails for linked hosts in 2.93 (everything above that is fine), so the behavior ends up being correct, it just prints an annoying error message in the terminal when hovering over the button.
```suggestion
        if is_linked(context.host):
```